### PR TITLE
render full screen on hover

### PR DESCRIPTION
**Pull request by justame**

### CODE REVIEW

**Comment by justame** [comment]
@lxgreen, please review

**Comment by oded-soffrin** [comment]
image

reproduce:
- load this https://rich-content-storybook-render-fullscreen-on-hover.surge.sh/?path=/story/plugins--image
- wait with the cursor outside of the viewer
- hover on image and click it (quickly)

also happen on master (https://rich-content-storybook-render-fullscreen-on-hover.surge.sh/?path=/story/plugins--image)
but only if you click when page is loading. this PR is making the issue worse IMO

**Comment by justame** [comment]
@oded-soffrin , It means that there is a problem with the logic. I'll review it with Alex and we will fix it.

File: e2e/tests/media.e2e.js [hunk 1]
```diff
@@ -343,13 +343,13 @@ describe('plugins', () => {
     after(() => cy.eyesClose());
 
     //TODO: fix this flaky test
-    it.skip('render upload modal', function() {
+    it('render upload modal', function() {
```
Comment (oded-soffrin) [inline e2e/tests/media.e2e.js:346]: @justame  did you intentionally restore those flaky tests ?

Comment (justame) [reply e2e/tests/media.e2e.js:346]: no, I only merged from master and pushed

**Pull request merged**